pkg/registries/registries.go

```diff
@@ -229,8 +231,7 @@ func (ms *mirrorSet) generateConfig(secretsConfig credentialprovider.DockerConfi
 		result += hostCapabilities
 		result += "\r\n"
 
-		// TODO: Remove this when we support TLS auth using CA certs - https://issues.redhat.com/browse/WINC-1291
-		result += "  skip_verify = true"
+		result += fmt.Sprintf("  ca = \"%s\"", strings.ReplaceAll(windows.TrustedCABundlePath, "\\", "\\\\"))
```

Comment:
Does this line need to be added for this to work?

Reply:
yes the bundle location needs to be specified